src/System.Device.Ports/System.Device.Ports.SerialPort/System.Device.Ports.SerialPort.csproj

```diff
@@ -0,0 +1,57 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.0;net6.0</TargetFrameworks>
+    <TargetFrameworks Condition="$(NETCoreSdkRuntimeIdentifier.StartsWith('win'))">$(TargetFrameworks);net6.0-windows10.0.17763.0</TargetFrameworks>
+    <LangVersion>9</LangVersion>
```

Comment:
NIT: Do you actually need this LangVersion or did you just copy this? If you don't need any features that you are not getting by default then let's just remove this for now.